src/axom/sidre/spio/IOManager.cpp

```diff
@@ -1161,6 +1161,13 @@ void IOManager::writeBlueprintIndexToRootFile(DataStore* datastore,
       for(IndexType i = 0; i < domain_to_file->getNumElements(); ++i)
       {
         domain_ids_array[i] = i;
+
+        // Skip if no rank is assigned
```

Comment:
Would this be cleaner by changing this to the following?
`if(domain_rank_map[i] >=0)
  {
     domain_file_map[i] = rank_file_map[domain_rank_map[i]);
  }`